dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/processor/queue/TaskResponsePersistThread.java

```diff
@@ -159,7 +159,11 @@ private boolean persist(TaskResponseEvent taskResponseEvent) {
                     }
                 }
                 TaskKillAckCommand taskKillAckCommand = new TaskKillAckCommand(ExecutionStatus.SUCCESS.getCode(), taskResponseEvent.getTaskInstanceId());
```

Comment:
It's better to check channel first, it is no need to create `taskKillAckCommand` when channel is null.

Reply:
Please review again.^_^